Objective-C/Internal/Replicator/CBLWebSocket.mm

```diff
@@ -41,8 +41,10 @@
 
 static constexpr size_t kMaxReceivedBytesPending = 100 * 1024;
 static constexpr NSTimeInterval kConnectTimeout = 15.0;
-static constexpr NSTimeInterval kIdleTimeout = 300.0;
 
+// The value should be greater than the heartbeat to avoid read/write timeout;
+// the current default heartbeat is 300 sec:
+static constexpr NSTimeInterval kIdleTimeout = 320.0;
```

Comment:
how hard would it be to reference the heartbeat from here and then multiply it by a ratio?  I'm guessing hard, otherwise you would have done it .. but just checking.